Publish workflow fails

```
Run dart pub get
  dart pub get
  shell: /usr/bin/bash -e {0}
  env:
    DART_HOME: /opt/hostedtoolcache/dart/3.2.6/x64
    PUB_CACHE: /home/runner/.pub-cache
    PUB_TOKEN: ***
Resolving dependencies...
Because smooth_sheets requires the Flutter SDK, version solving failed.

Flutter users should use `flutter pub` instead of `dart pub`.
Error: Process completed with exit code 1.
```

[The full log](https://github.com/fujidaiti/smooth_sheets/actions/runs/7687635283/job/20947888576)

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -5,8 +5,15 @@ on:
 
 jobs:
   publish:
+    name: Publish to pub.dev
+    runs-on: ubuntu-latest
     permissions:
       id-token: write
-    uses: dart-lang/setup-dart/.github/workflows/publish.yml@v1
-    with:
-      working-directory: package
+    steps:
+      - name: Install Flutter SDK
+        uses: subosito/flutter-action@v2
+        with:
+          channel: stable
+          flutter-version: 3.x
+      - run: flutter --version
+      - run: flutter pub publish --force

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -5,7 +5,7 @@ on:
 
 jobs:
   publish:
-    name: Publish to pub.dev
+    name: Publish
     runs-on: ubuntu-latest
     permissions:
       id-token: write
@@ -18,5 +18,9 @@ jobs:
         with:
           channel: stable
           flutter-version: 3.x
-      - run: flutter --version
-      - run: flutter pub publish --force
+
+      - name: Flutter version
+        run: flutter --version
+
+      - name: Publish to pub.dev
+        run: flutter pub publish --force

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -10,6 +10,9 @@ jobs:
     permissions:
       id-token: write
     steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
       - name: Install Flutter SDK
         uses: subosito/flutter-action@v2
         with:

